packages/qdrant-loader/tests/integration/core/worker/test_handler_orchestrator_integration.py

```
@@ -0,0 +1,96 @@
+import sys
+import types
+from datetime import UTC, datetime
+from pathlib import Path
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+
+from qdrant_loader.core.worker.handlers import IngestionJobHandler
+
+
+def _setup_qdrant_loader_core_stubs():
```

Ruff (0.15.13)

[warning] Missing return type annotation for private function `_setup_qdrant_loader_core_stubs`

Add return type annotation: `None`

(ANN202)